src/schema/v2/marketingCollections.ts

```diff
@@ -0,0 +1,371 @@
+import {
```

Comment:
Thinking to split this into `marketingCollection.ts` and `marketingCollections.ts`...

Reply:
Yea - up to you! Once this lands can do another pass at organization. This PR is getting pretty big (I had imagined there could be multiple iterative PR's here).